docs/guidelines/custom-log-rules.md

```diff
@@ -0,0 +1,3 @@
+# Custom Log Rules
+
+TODO:
```

Comment:
This file was referenced in some of the docs pages, but non-existent. So I created it. I'm assuming it's on a roadmap.